editor/trait/sanitize.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+trait Brizy_Editor_Trait_Sanitize {
+
+	public function sanitizeJson($data) {
+		if ( current_user_can( 'unfiltered_html' ) ) {
+			return $data;
+		}
+
+		if ( ! $dataDecoded = json_decode( $data, true ) ) {
+			return $data;
```

Comment:
Eu mai degraba as arunca aici exceptie.. pentru ca daca $data nu se deserializeaza inseamna ca jsonul e invalid.. si  asta ajunge in baza asa invalid... tu ce zici?